### ISSUE TITLE
Pins don't align with #define statements

### ISSUE THREAD

**Issue by ccwtruck** [gnea/grbl-Mega#174]
I am attempting to install GRBL-Mega 1.1g (the current release) on an ATmega2560R3, and try as i might, I get no motion on Y & Z axis. According to the cpu_map.h, I have the stepper drivers connected properly:
`#define X_STEP_BIT    2 // MEGA2560 Digital Pin 24
#define Y_STEP_BIT    3 // MEGA2560 Digital Pin 25
#define Z_STEP_BIT    4 // MEGA2560 Digital Pin 26

// Define step direction output pins. NOTE: All direction pins must be on the same port.
#define X_DIRECTION_BIT   7 // MEGA2560 Digital Pin 30
#define Y_DIRECTION_BIT   6 // MEGA2560 Digital Pin 31
#define Z_DIRECTION_BIT   5 // MEGA2560 Digital Pin 32
`
I have flashed with your .hex file several times, and compiled/ uploaded from source several times -- still, the Y and Z axis's show no sign of life.
This is on a CNC plasma table that worked well on the Uno with v1.1h.
I have double, triple & quadruple-checked my wiring -- it is correct; after all, i'm just upgrading the system board, and not re-wiring any more than that requires.
Please, can you tell me exactly which physical pins do my stepper controllers need to connect to?
Thank you in advance for any assistance!!!

**Comment by fra589** [comment]
Hi @ccwtruck  

Perhaps this doc: https://github.com/fra589/grbl-Mega-5X/wiki/Pinout-mapping-in-cpu_map.h will help you to find which pins are used...  
  
@++;  
Gauthier.

**Comment by ccwtruck** [comment]
Thank you for that; I've at least verified that my steppers are connected to the right pins. I am now wondering if my Y & Z steppers have taken a dump. They still light up, but with the connections being correct, that's the next logical link in the chain.

**Comment by ccwtruck** [comment]
After temporarily abandoning my Mega2560R3 project, I picked it back up the other day. I re-flashed the Mega with (first) @chamnit's latest version (which gave me X & Z motion) and now the version based on regular GRBL 1.1g by [Bacon8tor ](https://github.com/Bacon8tor/grbl-Mega-Edge-master/tree/main)   and still only had proper motion on both X & Z axis's. Y axis was still completely unresponsive, though -- even after I switched drivers, flashed GRBL several times, ran the EEPROM Clear sketch.... 
After much trial & error, suddenly -- and I do NOT know what I did -- I got Y axis to move, but in only one direction.  So, on a whim, I switched back to the original Makerbase driver board & fired it up again.  **Now, I have all three axis's moving in the proper directions.**
I cannot explain this -- perhaps a setting I changed, or a wire I moved -- but everything now works.
Now to start working on the torch height controller for the table.
